.github/workflows/washboard.yml

```diff
@@ -2,8 +2,9 @@ name: washboard
 
 on:
   push:
-    paths: 
+    paths:
       - 'washboard-ui/**'
+      - '.tool-versions'
```

Comment:
I don't see a problem with this being at the root level but will want one of our @wasmCloud/org-maintainers to confirm.

Reply:
Is there a downside to moving this under `washboard-ui`? I'd prefer not adding another file to the top level, if possible